src/main/java/Duke.java

```diff
@@ -1,10 +1,106 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
+    private static final String indentLine = "--------------------------------------------------";
+    private static Task[] tasks = new Task[100];
+    private static int numberOfTasks = 0;
+
+    public static void enterGreet() {
+        System.out.println(indentLine);
+        System.out.println("Hello! I'm Duke\n" + "What can I do for you?\n");
+        System.out.println(indentLine);
+    }
+
+    public static void farewellGreet() {
+        System.out.println("Bye. Hope to see you again soon!\n");
+        System.out.println(indentLine);
+    }
+
+    public static void Echo(String line) {
+        System.out.println(line);
+    }
+
+    public static void addList(String type, String input) {
```

Comment:
Perhaps a suggestion would be to group the addList and the displayList function into a sort of list class to improve the code quality